ml_genn/model.py

```diff
@@ -179,35 +179,35 @@ def evaluate_batched(self, data, time, save_samples=[]):
         spike_t       --  list of spike times for each sample index in save_samples
         """
 
+        n_complete = [0] * len(self.outputs)
         n_correct = [0] * len(self.outputs)
         accuracy = [0] * len(self.outputs)
         all_spikes = [[[] for i,_ in enumerate(self.layers)] for s in save_samples]
 
+        data_remaining = True
         data_iter = iter(data)
         batch_data, batch_labels = next(data_iter)
         batch_i = 0
 
-        batch_labels_queue = deque()
-
-        # Pad number of samples so pipeline can be flushed
-        pipeline_depth = self.calc_pipeline_depth()
-        pipeline_flush = False
-        pipeline_remaining = pipeline_depth
+        # Pipeline queues
```

Comment:
It might be worth clarifying either in the comment or the variable names that these are per-output depths and queues - I didn't _quite_ get it first time I read through